### DISCUSSION TITLE
[sdcard] How do I use an  external memory (SDcard) as a Notebook?

### DISCUSSION THREAD

**Discussion by AtlcatlQ** [gsantner/markor#1397] [Q&A] [answered]
I'm trying to set a folder in an external memory as the _Notebook_ (The main folder) but I just don't find the direction, help me, pls

**Comment by gsantner** [comment, marked as answer]
Go up (..) till you reach the /storage path, there other disks like your external sdcard appears.

Generally said, internal storage is recommended. Totally not recommending to set the notebook to SD-Card.   
The suggestion is instead to add the sdcard as favorite, and set "favorites" as startfolder for the filebrowser.

**Reply by AtlcatlQ** [reply]
Jaja, sorry, was a security thing in my phone. The app needs a permission to write in the sd card or in a folder in the sd card. So, I had to give the permissions manually (This is in case someone have the same trouble). 

This is what it showed when I tried to storage files in a folder without the permission:

![Screenshot_20210719_084220_net gsantner markor | width=50](https://user-images.githubusercontent.com/87624226/126183913-65d2a8d6-6ba7-4ee0-8f30-645e2908b287.jpg)

The strikethrough directions are the no permission to edit, isn't it?

![InkedScreenshot_20210719_083810_net gsantner markor_LI | width=50](https://user-images.githubusercontent.com/87624226/126182963-4a87ab82-baf3-4cf0-833c-2f1f0130cf60.jpg)

My phone is a Huawei Y9 prime 2019, for the documentation xd.

Sorry for my bad english

----

btw, I really love your app.

**Reply by gsantner** [reply]
I made some improvements for SD-Card support recently. Please see <https://github.com/gsantner/markor/issues/603>.
